crates/ruff_linter/src/rules/airflow/rules/function_signature_change_in_3.rs

```diff
@@ -0,0 +1,117 @@
+use crate::checkers::ast::Checker;
+use crate::rules::airflow::helpers::FunctionSignatureChangeType;
+use crate::{FixAvailability, Violation};
+use ruff_macros::{ViolationMetadata, derive_message_formats};
+use ruff_python_ast::{Arguments, Expr, ExprAttribute, ExprCall};
+use ruff_python_semantic::Modules;
+use ruff_python_semantic::analyze::typing;
+use ruff_text_size::Ranged;
+
+/// ## What it does
+/// Checks for Airflow function calls that will raise a runtime error in Airflow 3.0
+/// due to function signature changes, such as functions that changed to accept only
+/// keyword arguments, parameter reordering, or parameter type changes.
+///
+/// ## Why is this bad?
+/// Airflow 3.0 might introduce changes to function signatures. Code that
+/// worked in Airflow 2.x may raise a runtime error if not updated in Airflow
+/// 3.0.
```

Comment:
What do you think about something like this?

```suggestion
/// Airflow 3.0 introduces changes to function signatures. Code that
/// worked in Airflow 2.x will raise a runtime error if not updated to work with Airflow
/// 3.0.
```

It seems like some function signatures have changed since y'all are adding this rule :)